src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/HearingService.java

```diff
@@ -30,10 +30,13 @@
 @RequiredArgsConstructor
 public class HearingService {
 
+    DateTimeFormatter dateFormatter = DateTimeFormatter.ofPattern("d MMM yyyy");
+
     static final String TOP_LEVEL_HEARING_ID = "00000000-0000-0000-0000-000000000000";
 
     // Helper class to handle sorting by date, time, and type.
-    private record HearingSortingKey(LocalDate hearingDate, String hearingTime, String hearingType) implements Comparable<HearingSortingKey> {
+    private record HearingSortingKey(LocalDate hearingDate, String hearingTime,
+                                     String hearingType) implements Comparable<HearingSortingKey> {
```

Comment:
eslint auto changed this on save

Reply:
You can't blame eslint for doing that...only the person who switched eslint on ;oP